### DISCUSSION TITLE
Add an HTTP/2 reserve proxy in front of Home Assistant

### DISCUSSION THREAD

**Discussion by usingapc** [hassio-addons/addon-nginx-proxy-manager#329] [Q&A] [answered]
I have 25 cameras on my Ubiquiti UNVR-pro that I would like to use in Home Assistant with the Unfi Protect Add-on. The problem I am having is that if I don't pre-load the stream, I get a minimum of a 30 second delay. If I preload the stream on all the cameras the entire system runs very slowly on a Raspberry Pi 4 with 4GB ram. I read on the Add-on documentation about enabling LL-HLS in the stream integration and that helped slightly. There is also a note about using NGINX Proxy Add-on to put an HTTP/2 reserve proxy in front of Home Assistant. I can't find any instructions on how to do that. Can anyone help me figure out how to do that?

**Comment by frenck** [comment, marked as answer]
> I can't find any instructions on how to do that. 

If you won't know what it entails, you should probably just not do that.
However, how to install and use this add-on is documented in the add-on documentation of this add-on.
